File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var smallerNumbersThanCurrent = function(nums) {
    let result=[];
    for(let i=0;i<nums.length;i++){
        let numSmall=0;
        for(let j=0;j<nums.length;j++){
            if(nums[i]>nums[j]){
               numSmall++
            }
        }
        result.push(numSmall);
    }
    return result;
};

File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.cs
public class Solution {
    public int[] SmallerNumbersThanCurrent(int[] nums) {
        List<int> result=new List<int>();
        for(int i=0;i<nums.Length;i++){
            int numSmall=0;
            for(int j=0;j<nums.Length;j++){
                if(nums[i]>nums[j]){
                    numSmall++;
                }
            }
            result.Add(numSmall);
        }
        return result.ToArray();
    }
}